### PR TITLE
Bring back the HTTP console commands

### LINKED ISSUE #35879: Quarkus 3.4.0.CR1 does not have HTTP commands in dev mode
### Describe the bug

When running in dev mode there are no longer HTTP options like opening the web app and dev ui
3.4.0.CR1 has following 
image

while 3.2.2 has
image

### Expected behavior

HTTP commands are available in 3.4.0

### Actual behavior

no HTTP commands in 3.4.0

### How to Reproduce?

Run any project that has HTTP endpoints like REST in dev mode and print help of dev mode. Or try to use `d` or `w` to open dev ui or the web app

### Output of `uname -a` or `ver`

Darwin  22.6.0 Darwin Kernel Version 22.6.0

### Output of `java -version`

openjdk version "17.0.2" 2022-01-18

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

3.4.0.CR1

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

apache-maven-3.8.4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by geoand**
These were erroneously removed in 19aae16c113dbc05e3fc4808719a5ba9dfd37ade

Fixes: #35879

### CODE REVIEW

File: extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/console/ConsoleProcessor.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        }
+        if (context == null) {
+            context = ConsoleStateManager.INSTANCE.createContext("HTTP");
+        }
+        Config c = ConfigProvider.getConfig();
+        String host = c.getOptionalValue("quarkus.http.host", String.class).orElse("localhost");
+        String port = c.getOptionalValue("quarkus.http.port", String.class).orElse("8080");
+        context.reset(
+                new ConsoleCommand('w', "Open the application in a browser", null, () -> openBrowser(rp, np, "/", host, port)),
+                new ConsoleCommand('d', "Open the Dev UI in a browser", null, () -> openBrowser(rp, np, "/q/dev", host, port)));
```
Comment (gsmet) [inline extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/console/ConsoleProcessor.java:36]: @phillip-kruger can you make sure this the URL we want here? Ideally, we should avoid having a redirect.

Comment (phillip-kruger) [reply extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/console/ConsoleProcessor.java:36]: Yes probably best to go directly to /q/dev-ui

Comment (geoand) [reply extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/console/ConsoleProcessor.java:36]: Good point, I'll fix it

**Comment by geoand** [comment]
PR updated

**Review by phillip-kruger — APPROVED** [review]
Thanks!

**Pull request merged**